unison-cli/src/Unison/Codebase/Editor/HandleInput/Update2.hs

```diff
@@ -135,6 +135,9 @@ handleUpdate2 = do
       then pure tuf
       else do
         Cli.respond Output.UpdateStartTypechecking
+        currentPath <- Cli.getCurrentPath
+        currentBranch0 <- Cli.getBranch0At currentPath
+        let namesIncludingLibdeps = Branch.toNames currentBranch0
```

Comment:
I think shadowing the above definition here for a different purpose is going to be pretty confusing;

I'd suggest naming one of them `projectNamesIncludingLibdeps`;

but at a glance I'm not actually sure whether you ever actually want the names at the `update-in` path, you might just always want the project-root names 🤔

Reply:
Naming one of them that, or naming both of them that?  Also, this code currently would run in loose code also until we eliminate loose code.